WiDS_2019_Cognite_Part1.ipynb

```diff
@@ -7,7 +7,7 @@
     "id": "eAXnMj5E9jWG"
    },
    "source": [
-    "<a href=\"https://colab.research.google.com/github/cognitedata/WiDS-2019/blob/master/WiDS_2019_Cognite_Part1.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>\n",
+    "<a href=\"https://colab.research.google.com/github/cognitedata/WiDS-2019/blob/r2_score/WiDS_2019_Cognite_Part1.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>\n",
```

Comment:
I think the link should be still `master`. After merging this into master, the old should reflect the changes.

Reply:
Yes, thanks. I changed it.